_job-scheduler-plugin/index.md

```diff
@@ -0,0 +1,130 @@
+
+```java
+public class SampleExtensionRestHandler extends BaseRestHandler {
+    public static final String WATCH_INDEX_URI = "/_plugins/scheduler_sample/watch";
+```
+
+You then need to define the job configuration by [extending](https://github.com/opensearch-project/job-scheduler/blob/main/sample-extension-plugin/src/main/java/org/opensearch/jobscheduler/sampleextension/SampleJobParameter.java) `ScheduledJobParameter`. You can also define the fields used by your plugin like `indexToWatch` as shown in the [example](https://github.com/opensearch-project/job-scheduler/blob/main/sample-extension-plugin/src/main/java/org/opensearch/jobscheduler/sampleextension/SampleJobParameter.java) `SampleJobParameter` file. This job configuration will be saved as a document in an index defined by you as shown in this [example](https://github.com/opensearch-project/job-scheduler/blob/main/sample-extension-plugin/src/main/java/org/opensearch/jobscheduler/sampleextension/SampleExtensionPlugin.java#L54).
+
+## Parameters configuration
+
+You can configure your plugin's parameters by referencing and modifying the [example](https://github.com/opensearch-project/job-scheduler/blob/main/sample-extension-plugin/src/main/java/org/opensearch/jobscheduler/sampleextension/SampleJobParameter.java) `SampleJobParameter.java` file to fit your needs:
+
+```java
+/**
+ * A sample job parameter.
+ * <p>
+ * It adds an additional "indexToWatch" field to {@link ScheduledJobParameter}, which stores the index
+ * the job runner will watch.
+ */
+public class SampleJobParameter implements ScheduledJobParameter {
+    public static final String NAME_FIELD = "name";
+    public static final String ENABLED_FILED = "enabled";
+    public static final String LAST_UPDATE_TIME_FIELD = "last_update_time";
+    public static final String LAST_UPDATE_TIME_FIELD_READABLE = "last_update_time_field";
+    public static final String SCHEDULE_FIELD = "schedule";
+    public static final String ENABLED_TIME_FILED = "enabled_time";
+    public static final String ENABLED_TIME_FILED_READABLE = "enabled_time_field";
+    public static final String INDEX_NAME_FIELD = "index_name_to_watch";
+    public static final String LOCK_DURATION_SECONDS = "lock_duration_seconds";
+    public static final String JITTER = "jitter";
+
+    private String jobName;
+    private Instant lastUpdateTime;
+    private Instant enabledTime;
+    private boolean isEnabled;
+    private Schedule schedule;
+    private String indexToWatch;
+    private Long lockDurationSeconds;
+    private Double jitter;
+```
+
+Next, build out your parameters from the declared variables. The following sample shows the request parameters to set while building your plugin:
```

Comment:
Is there another way to phrase "build out your parameters from the declared variables" that might be a little more intuitive?

Reply:
rephrased this to be more descriptive and more intuitive